src/runmode-unix-socket.c

```diff
@@ -831,6 +831,16 @@ TmEcode UnixSocketDatasetLookup(json_t *cmd, json_t *answer, void *data)
     }
 }
 
+static bool json_u32_value(json_t *jarg, uint32_t *ret)
```

Comment:
style is off

Reply:
changing it to `JsonU32Value`, I was imitating `json_integer_value` style